### ISSUE TITLE
vConsole is failing in Next.js app router

### ISSUE THREAD

**Issue by kristofferso** [needle-tools/needle-engine-support#189]
### Describe the bug 💬

When running a Next.js project with the app router in dev I get the following error when reloading the page on mobile or when the browser simulates a mobile. I suspect it's because vConsole only loads when the screen size is less than a certain amount. 

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'length')
    at vconsole.min.js:10:221845
```

Next.js version 14.0.3

### Operating System 👩‍💻

macOS

### What browsers are you seeing the problem on? 🏄‍♂️

Chrome

### Editor Version 🎲

2020.3

### Needle Exporter version

3.31

### Steps to reproduce 🔢

```bash
1. Run `npm run dev`
2. Enter device preview mobile from developer console
3. Refresh page
4. See that the scene is stuck at loading because vconsole fails
...
```

### Validations 🩹

- [X] Follow our [Code of Conduct](https://github.com/needle-tools/needle-engine-support/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://docs.needle.tools).
- [X] Check that there isn't [already an issue](https://github.com/needle-tools/needle-engine-support/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Needle-Engine-specific issue. For example if it's a Unity related bug, it should be reported to [Unity](https://unity3d.com/unity/qa/bug-reporting) instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/needle-tools/needle-engine-support/discussions) or join our [Discord Chat Server](https://discord.needle.tools).
- [ ] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by marwie** [comment]
Do you see this error in vconsole? Can you check the stacktrace? The error might be misleading

**Comment by kristofferso** [comment]
No I can't see anything in vconsole because it doesn't load. 

The stack trace is just 60+ repeats of: 
```
Promise.then (async)
readData @ three.module.js:42216
eval @ three.module.js:42235
```

**Comment by marwie** [comment]
Is it the same in the regular browser console? Can you see what is evaluated there?

**Comment by kristofferso** [comment]
This is from the browser console. The virtual console doesn't load. The same error happens in your Next.js starter project.

**Comment by marwie** [comment]
Hi, I just tried with [our template](https://github.com/needle-engine/nextjs-sample) and I don't get the error there
Do you refer to a different starter project?

**Comment by kristofferso** [comment]
Right, I should have specified in my last comment that this is for the app router. I made a fork of the repo that reproduces the error: https://github.com/kristofferso/needle-test/tree/main/src
